lib/src/_utils.dart

```diff
@@ -9,5 +9,5 @@ extension ObjectX on Object {
   /// Casts the current object to type [T].
   ///
   /// In some production compilers, this cast is skipped.
-  T unsafeCast<T>() => this; // ignore: return_of_invalid_type
+  T unsafeCast<T>() => this as T; // ignore: return_of_invalid_type
```

Comment:
nit: Thanks!

To keep this functionality, I needed to add the following pragmas:
```
  @dart2js.tryInline
  @pragma('dart2js:as:trust')
```